src/models/tiles/tile-component-info.ts

```diff
@@ -20,6 +21,7 @@ export interface ITileComponentInfo {
 const gTileComponentInfoMap = new Map<string, ITileComponentInfo>();
 
 export function registerTileComponentInfo(tileComponentInfo: ITileComponentInfo) {
+  registerPlugins();
```

Comment:
Instead of putting it here. 
I'd recommend putting it in `text-registration.ts` just before it calls `registerTileComponentInfo`.